modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/wstep/impl/CertificateEnrollmentServiceImpl.java

```diff
@@ -112,6 +111,17 @@ public void requestSecurityToken(String tokenType, String requestType,
 	                                 Holder<RequestSecurityTokenResponse> response) throws
 	                                 WindowsDeviceEnrolmentException {
 
+		String headerBinarySecurityToken = null;
+		List<Header> ls = getHeaders();
+		for (int i = 0; i < ls.size(); i++) {
+			Header headerElement = ls.get(i);
+			String nodeName = headerElement.getName().getLocalPart();
+			if (nodeName.equals(Constants.SECURITY)) {
```

Comment:
Swap the two parameters compared against each other to avoid redundant null checks.